examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -83,14 +83,17 @@ CommissioningParameters PairingCommand::GetCommissioningParameters()
     return CommissioningParameters();
 }
 
-CHIP_ERROR PairingCommand::PairWithQRCode(NodeId remoteId)
+CHIP_ERROR PairingCommand::PairWithCode(NodeId remoteId)
 {
-    return CurrentCommissioner().PairDevice(remoteId, mOnboardingPayload);
-}
-
-CHIP_ERROR PairingCommand::PairWithManualCode(NodeId remoteId)
-{
-    return CurrentCommissioner().PairDevice(remoteId, mOnboardingPayload);
+    if (mNetworkType == PairingNetworkType::None)
```

Comment:
So just to check, what codepath will `chip-tool pairing qrcode node-id code` as used in our CI (because we are doing QRCode-based on-network commissioning) take here?  Seems to me like it will take the EstablishPASEConnection path?  If not, why not?  And if it does, what ends up doing the commissioning eventually?

We can change what we run in CI as needed, of course, but I'm not sure whether what we do now will keep working and if not what we should do instead.

Reply:
(don't know if this comment was from before the next commit or after)

I noticed the problem with the CI and added some more pairing commands to specify the pase only connection specifically. I THINK that makes it easier, but I'll have to see once the CI runs through.

The discussion of the chip-tool command line arguments and what we're expecting there is somewhat separate from the discussion of the SDK api because I think we're allowed to update chip tool post TE8 branch if we later decide to make that tool more verbose or add options. As long as I can get CI to pass, I don't have any strong opinions on what the chip-tool supports.

Reply:
> (don't know if this comment was from before the next commit or after)

Before.  I agree that the key part here is not breaking CI and it was not here how the initial PR was avoiding that.  The new changes make more sense there.